src/main/daml/Daml/Finance/Claims/Lifecycle/Rule.daml

```diff
@@ -114,7 +114,7 @@ template Rule
           let
             electionData = Some (v.electorIsOwner, v.claim)
             currentKey = BaseInstrument.getKey $ toInterface claimInstrument
-            newKey = currentKey with version = sha256 $ show remaining
+            newKey = currentKey with version = sha256 $ mconcat [show electionTime, show remaining]
```

Comment:
you could do
newKey = currentKey with version = sha256 . mconcat $ show <$> [electionTime, remaining]

Reply:
I tried that, but `electionTime` and `remaining` are of different types, so the compiler did now allow it.